<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pardus Testing Framework</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>451</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>20</red>
         <green>19</green>
         <blue>18</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>20</red>
         <green>19</green>
         <blue>18</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QTextEdit" name="text_edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>561</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="package_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>464</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Did the above test work as expected?</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QRadioButton" name="no_button">
      <property name="text">
       <string>&amp;No</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Enter your observation: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QRadioButton" name="yes_button">
      <property name="text">
       <string>&amp;Yes</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="text_observation">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>531</y>
     <width>569</width>
     <height>70</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>64</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_button">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>609</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>N&amp;ext</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>609</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Finish</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>yes_button</tabstop>
  <tabstop>no_button</tabstop>
  <tabstop>text_observation</tabstop>
  <tabstop>next_button</tabstop>
  <tabstop>clear_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>yes_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>text_observation</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>no_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>text_observation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_button</sender>
   <signal>clicked()</signal>
   <receiver>text_observation</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>495</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>no_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yes_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quit_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>609</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>608</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
